lib/wfsc/falco_wfsc_loop.m

```diff
@@ -12,11 +12,19 @@
 fprintf('\nBeginning Trial %d of Series %d.\n', mp.TrialNum, mp.SeriesNum);
 mp.thput_vec = zeros(mp.Nitr+1, 1);
 
+flagBreak = false;
+
 for Itr = 1:mp.Nitr
     
     
     %% Bookkeeping
-    fprintf(['WFSC Iteration: ' num2str(Itr) '/' num2str(mp.Nitr) '\n' ]);
+    fprintf(['WFSC Iteration: ' num2str(Itr) '/' num2str(mp.Nitr) ', ' datestr(now) '\n' ]);
+    
+    % update subdir for scicam images
+    if isfield(mp, 'tb')

```

Comment:
@davidsmarx Please move this block to be within `falco_plot_progress_omc.m`

Reply:
falco_plot_progress is too late. We want to update the subdir name for this iteration. Adding 'omc' keyword in the if statement.